canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html

```diff
@@ -16,17 +16,19 @@
   {% endif %}
 {% endblock stylesheets %}
 
-{% block content %}
 
+{% block breadcrumb %}
+<nav>
+    <h3>
```

Comment:
Can we make this respect the header type (h1) and classes that UX provided? (see removed lines 25-27 below)